e2e-tests/portal/cypress/integration/journeys/dashboard/dashboard-deals/dashboard-deals-create-deal.spec.js

```diff
@@ -0,0 +1,49 @@
+const relative = require('../../../relativeURL');
+const mockUsers = require('../../../../fixtures/mockUsers');
+
+const { dashboard, selectScheme } = require('../../../pages');
+
+const BANK3_GEF_MAKER1 = mockUsers.find((user) => (user.roles.includes('maker') && user.username === 'BANK3_GEF_MAKER1'));
+const BANK1_MAKER1 = mockUsers.find((user) => (user.roles.includes('maker')));
+
+context('Create application as gef-only bank and other bank', () => {
+  describe('Create application as gef-only bank', () => {
+    before(() => {
+      Cypress.Cookies.preserveOnce('connect.sid');
+      cy.login(BANK3_GEF_MAKER1);
+      dashboard.visit();
+      cy.url().should('eq', relative('/dashboard/deals/0'));
+    });
+
+    it('When clicking Create new button, it takes user straight to mandatory criteria page', () => {
+      dashboard.createNewSubmission().click();
+      cy.url().should('eq', relative('/gef/mandatory-criteria'));
+    });
+
+    it('When clicking continue, it should lead to name your application', () => {
+      dashboard.mandatoryCriteriaYes().click();
+      dashboard.continueButton().click();
+      cy.url().should('eq', relative('/gef/name-application'));
+    });
+  });
+
+  describe('Create application as all other banks', () => {
```

Comment:
Could we rename descriptions so it's clearer?

```
context('Create application as GEF-only bank and GEF-and-BSS bank', () => {
...


describe('Create application as GEF-and-BSS bank', () => {

```
Just "all other banks" isn't super clear

Reply:
done